tinytodo-go/internal/app/server/entitystore/list.go

```diff
@@ -0,0 +1,96 @@
+package entitystore
+
+import (
+	"code.byted.org/binaryauthorization/tinytodo-go/internal/app/server/entitystore/entitytype"
+	"code.byted.org/binaryauthorization/tinytodo-go/internal/app/server/entitystore/taskstate"
+	"github.com/cedar-policy/cedar-go"
+	"strconv"
+)
+
+// ListUID is a transparent wrapper around EntityUID, to make it clear that we want a List's EntityUID.
+//
+// Note that we use inheritance instead of alias, because we want to inherit methods. See [blog post].
```

Comment:
Tiny nit: this is "composition" or "embedding" rather than "inheritance".

Reply:
Thanks for spotting the mistake, will change this